src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,58 @@
+import { DraftClaimantResponse } from 'features/claimant-response/draft/draftClaimantResponse.ts'
+import { ResponseRejection } from 'claims/models/response/core/responseRejection.ts'
+import { FormaliseRepaymentPlanOption } from 'claimant-response/form/models/formaliseRepaymentPlanOption'
+import { CourtDetermination } from 'claims/models/response/core/courtDetermination.ts'
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+
+export class ClaimantResponseConverter {
+
+  public static covertToClaimantRespose (draftClaimantResponse: DraftClaimantResponse): any {
+
+    if (draftClaimantResponse.rejectionReason && draftClaimantResponse.rejectionReason.text.length > 0) {
+      return {
+        'type': 'rejection',
+        'amountPaid': 0,
+        'responseRejection': this.createResponseRejection(draftClaimantResponse)
+      }
+    } else {
+      return {
+        'type': 'acceptation',
+        'amountPaid': draftClaimantResponse.paidAmount ? draftClaimantResponse.paidAmount : 0,
+        'courtDetermination': this.createCourtDetermination(draftClaimantResponse),
+        'formaliseOption': this.getFormaliseOption(draftClaimantResponse.formaliseRepaymentPlan.option)
+      }
+    }
+  }
+
+  public static createResponseRejection (draftClaimantResponse: DraftClaimantResponse): ResponseRejection {
```

Comment:
Make private unless it has to be public. Please apply the same to other functions that does not have to be exposed / are not used by other classes.

Reply:
well ...since it is more or less a static utility method, I felt it might be reused in other places. I didn't see any benefit in making it private. I am happy to change it to private if that makes more sense, please let me know.